### DISCUSSION TITLE
Structure formula question.

### DISCUSSION THREAD

**Discussion by CK-CHO** [Open-Systems-Pharmacology/Forum#800] [Questions & Problems] [answered]
I have a two questions about structure formula.

First, i didn't find biliary clearance formulation in the Mobi software although i incorporated the renal and biliary clearance both in the model. I only found the formula of renal clearance 
(dN/dt=V*CLspec*C_pls*fu) i wonder that same equation is applied in biliary clearance. if not, where can i find the biliary clearance formula.

Second, i confused the specific clearance formula 
fu * (Q * (1 - HCT) - CL_pls * BW)>0 ? CL_pls * BW * Q * (1 - HCT) / (fu * (Q * (1 - HCT) - CL_pls * BW) * V) : 1000000

In above formula, is "fu * (Q * (1 - HCT) - CL_pls * BW)>0 ?" part indicates condition of this formula?
(i understood that only  fu * (Q * (1 - HCT) - CL_pls * BW)>0 condition is appropriate in the model)
Also, "CL_pls * BW * Q * (1 - HCT) / (fu * (Q * (1 - HCT) - CL_pls * BW) * V) : 1000000" part, i wonder that ":1000000" what is this.

I'm sorry to confuse you by not influent english skill.
Thank you. 

![question figure 1](https://user-images.githubusercontent.com/49744058/114351321-65842400-9ba5-11eb-8362-7539ade84efe.png)

![question figure 2](https://user-images.githubusercontent.com/49744058/114351008-f9092500-9ba4-11eb-859e-5df6f4a702cd.png)

**Comment by PavelBal** [comment, marked as answer]
Hi,

1) The biliary transport is an active transporter, located in the molecules building-block

![image](https://user-images.githubusercontent.com/25903040/114354715-e3790880-9b6e-11eb-9cd3-952c72963730.png)

2) Yes, it is a conditional formula of type

`Equation1 ? Equation2 : Equation3`

meaning "If Equation1 is true, then Equation2, otherwise Equation3".

In this particular case, `Equation1` checks if the blood flow rate to the kidney is larger than the entered plasma clearance. If it is true, the blood flow rate is **not** the limiting factor and `Equation2` is used, which calculates the plasma clearance. If the blood flow rate is lower than the entered plasma clearance (perfusion limited case), `Equation3` which is just an arbitrary high value.

**Comment by CK-CHO** [comment]
Dear PavelBal

Thank you for your rapid and kindness answers. your answers will be of great help to our future research.
